packages/algolia/src/client.ts

```diff
@@ -39,6 +48,14 @@ export class AlgoliaSearchClient {
     await this.index.saveObject(AlgoliaSearchClient.getAlgoliaObject(payload));
   }
 
+  async saveEntity({ id, type, data }: Entity): Promise<void> {
```

Comment:
Why can't we use save?

Reply:
I could - this needs to be refactored.
We shouldn't check entity by field "existence".
We should provide it explicitly.

Reply:
and why is that? with this method now am i able to save an entity 'research-output' with any arbitrary data? looks like it - judging by the type https://github.com/yldio/asap-hub/blob/56f9d41f8bfc9ca8aea35fee861c859ac60bfc11/packages/algolia/src/client.ts#L39
I think we shouldnt be allowing for any random entity data to be stored and remove type-safety like that - if we are going to we might as well use algolia client directly